src/rover_control/rover_control.hpp

```diff
@@ -18,6 +18,13 @@ class Drive {
 		void set_angle(float angle);
 		void set_movement(float speed, float angle);
 
+		static float actual_left_speed;
+		static float actual_right_speed;
+		static ::drive::DriveMode_Mode actual_drive_mode;
+		static std::chrono::steady_clock::time_point last_update_received;
```

Comment:
Instead of making these static, you can capture `this` in the lambda function's capture clause (the square brackets: `[this](...)`. Then you can access the object's member variables in the function definition